guides/tutorial/part-2/index.md

```diff
@@ -1,17 +1,15 @@
-<!-- Heads up! This is a generated file, do not edit directly. You can find the source at https://github.com/ember-learn/super-rentals-tutorial/blob/master/src/markdown/tutorial/part-2/index.md -->
+Super, vous avez atteint la seconde partie du tutoriel&nbsp;! Dans les sections suivantes, nous continuerons à construire sur la base des concepts clés appris dans la première partie.
 
-Hooray, you've made it to the second part of the tutorial! In the following sections, we'll build on the core concepts that we learned in the first part of the tutorial.
+En cours de route, nous ajouterons de nouvelles fonctionnalités à notre app "Super Rentals". À la fin de cette section, vous aurez implémenté une fonction de recherche et refactoré une bonne partie du code pour utiliser de nouveaux concepts Ember.
 
-Along the way, we'll also add some new features to our Super Rentals app. By the end of this section, we'll have implemented some search functionality and refactored a good bit of our code to use some new Ember concepts
+<img src="/images/tutorial/part-2/provider-components/filtered-results@2x.png" alt="Fonctionnalité de recherche de l'app Super Rentals" width="1024" height="797">
 
-<img src="/images/tutorial/part-2/provider-components/filtered-results@2x.png" alt="Search functionality in the Super Rentals app" width="1024" height="797">
+Dans cette deuxième partie, nous couvrirons les concepts suivants&nbsp;:
 
-In part two, we'll cover the following concepts:
-
-- Dynamic segments
-- Ember services
+- Les segments dynamiques
+- Les services Ember
 - Ember Data
-- Adapters and serializers
-- The provider component pattern
+- Adaptateurs et sérialiseurs
+- Le patron de conception _provider component_
```

Comment:
The French translation sounds really weird, I've never heard "Il faudrait utiliser le patron de conception (...)".  I would keep the Engligh sentence mostly as is, which is (to me) way more commonly used than the French equivalent. 

What about:

```suggestion
- Le pattern _provider component_
```

what about:

```suggestion
- Le modèle _provider component_
```

what about:

```suggestion
- La conception de composants dit _provider component_
```

Reply:
In this sentence, pattern means "design pattern", it can be translated by: 
- Modèle de conception
- Patron de conception
- Motif de conception

I advocate against "modèle" because I think it can be very confusing in the Ember context, with Ember data models that we translate "modèle de données".

The second most used term is "Patron", I have a DUT in programming background and I am personally familiar with it, I shouldn't be the only one. But I rarely heard "Motif", so I would go with "patron".